aspnetcore/mvc/views/partial.md

```diff
@@ -1,112 +1,152 @@
 ---
-title: Partial Views in ASP.NET Core
+title: Partial views in ASP.NET Core
 author: ardalis
 description: Learn how a partial view is a view that's rendered within another view and when they should be used in ASP.NET Core apps.
 ms.author: riande
-ms.date: 03/14/2018
+ms.custom: mvc
+ms.date: 07/02/2018
 uid: mvc/views/partial
 ---
-# Partial Views in ASP.NET Core
+# Partial views in ASP.NET Core
 
 By [Steve Smith](https://ardalis.com/), [Maher JENDOUBI](https://twitter.com/maherjend), [Rick Anderson](https://twitter.com/RickAndMSFT), and [Scott Sauber](https://twitter.com/scottsauber)
 
-ASP.NET Core MVC supports partial views, which are useful when you have reusable parts of web pages you want to share between different views.
+ASP.NET Core MVC supports partial views, which are useful for sharing reusable parts of web pages across different views.
```

Comment:
Consider adding --
For example, the default templates for a new web app with individual user accounts creates the following partials:

* _LoginPartial.cshtml:  Called in *_Layout.cshtml*.
* _CookieConsentPartial: Called in *_Layout.cshtml*.
*  *_ValidationScriptsPartial.cshtml*: ...